environments/kubernetes/manifests/thanos-receive-statefulSet-default.yaml

```diff
@@ -109,10 +109,6 @@ spec:
           requests:
             cpu: 32m
             memory: 16Mi
-      volumes:
-      - configMap:
-          name: observatorium-tenants-generated
-        name: observatorium-tenants
```

Comment:
We still want this, no?

Reply:
I overrode the volume claims that comes from kube-thanos to fix this issue.